packages/google_maps_flutter/google_maps_flutter_ios/example/ios11/ios/RunnerTests/FLTGoogleMapJSONConversionsConversionTests.m

```diff
@@ -287,4 +287,18 @@ - (void)testCameraUpdateFromChannelValueZoomTo {
   [classMockCameraUpdate stopMocking];
 }
 
+- (void)testLengthsFromPatterns {
+  NSArray *patterns = @[ @[ @"gap", @10 ], @[ @"dash", @10 ] ];
```

Comment:
can you use diff numbers for them?

Reply:
do they have to be ints?

Reply:
Not necessarily. According to the docs it's an array of `NSNumber*` , so floats should be fine too

Reply:
can you also use floating values. just wanna make sure that works too